examples/python/pytorch/resnet152_DDP_training.py

```diff
@@ -0,0 +1,95 @@
+import os
+import tempfile
+import torch,torchvision
+import torch.distributed as dist
+import torch.nn as nn
+import torch.optim as optim
+import argparse
+import torch.multiprocessing as mp
+from resnet_torch import resnet152
+import torchvision.transforms as transforms
+import time
+
+
+def setup(rank, world_size):
+    os.environ['MASTER_ADDR'] = 'localhost'
+    os.environ['MASTER_PORT'] = '12355'
+    dist.init_process_group("nccl", rank=rank, world_size=world_size)
+
+def cleanup():
+    dist.destroy_process_group()
+
+def train(gpu, args):
```

Comment:
I would change arg "gpu" to "local_gpu_id" or even better "local_rank". We use local rank and global rank as standard term for a node's rank in a distributed training.